src/components/data/LocalVariablesContent.js

```diff
@@ -85,9 +115,22 @@ const LocalVariablesContent = ({ workspace, workspace: { workspace: { googleProj
       Utils.convertValue(newBaseType, editValue)
 
     await Ajax().Workspaces.workspace(namespace, name).shallowMergeNewAttributes({ [editKey]: parsedValue })
+    const originalDescription = attributes[toDescriptionKey(originalKey)]
+    if (originalDescription || editDescription) {
```

Comment:
I like having comments when there's a few if statements in a block. Something like "When editing description, do X. When deleting a description, do Y." This becomes helpful when readers are figuring out what actions enter which if blocks. 

Btw, can we minimize the number of ajax calls we're making? For example, I feel like it should be possible to edit the key and description of a row in just one ajax call. Am I wrong about that?

Reply:
* Fixed the duplicate value issue - turns out I should have returned null instead of attempting to renderCellContents.
* Totally restructured the saveAttribute logic, adding comments where applicable - your key-only update issue should be resolved.
* We now make only two Ajax calls - one to delete and one to merge attributes.